conf/nginx/server.tpl.conf

```diff
@@ -39,8 +39,12 @@ server {
 }
 
 server {
+    {% if sni_forward_enabled != "True" %}
     listen 443 ssl http2;
     listen [::]:443 ssl http2;
+    {% else %}
+    listen 127.0.0.1:443 ssl http2;
```

Comment:
Use another port than 443 to avoid conflict with 0.0.0.0:443 ?

Reply:
Or maybe use a socket ?

Reply:
(moved to port 444, flagging as resolved)